itou/job_applications/models.py

```diff
@@ -432,6 +433,34 @@ def can_update_hiring_start(self):
             JobApplicationWorkflow.STATE_POSTPONED,
         ]
 
+    # Job application freshness
+
+    @cached_property
+    def last_state_change(self):
+        if not self.logs:
+            return None
+
+        return self.logs.most_recent()
+
+    @cached_property
+    def is_old(self):
+        """
+        A job application is considered to be old when:
```

Comment:
Pinaillage : `s/is considered to be old/is considered old/`

Reply:
Je change, merci !